hogben/utils.py

```diff
@@ -138,49 +138,29 @@ def __corner(self, results):
         axes[self.ndim-1, self.ndim-1].set_xlabel(self.params[-1].name)
         return fig
 
-def fisher(qs, xi, counts, models, step=0.005):
+def fisher(qs: list, xi: list, counts: list, models: list, step: float=0.005):
     """Calculates the Fisher information matrix for multiple `models`
        containing parameters `xi`.
 
     Args:
-        qs (list): Q points for each model.
+        qs (list[list]): Q points for each model.
```

Comment:
I have recently gone from typing in the docstring to typing in the arguments list. I think we should only do it in the args, and not have reprtition (also as it stands these two disagree with eachother!)